servers/rendering/renderer_scene_cull.cpp

```diff
@@ -958,6 +958,8 @@ void RendererSceneCull::instance_set_blend_shape_weight(RID p_instance, int p_sh
 	if (instance->mesh_instance.is_valid()) {
 		RSG::mesh_storage->mesh_instance_set_blend_shape_weight(instance->mesh_instance, p_shape, p_weight);
 	}
+
+	_instance_queue_update(instance, true, true);
```

Comment:
I don't think we need to update the AABB or dependencies, do we? Setting these to true is kind of expensive. So they should be false unless needed

Reply:
Tested locally, we don't need to update the  AABB or dependencies, but I wonder if we should update the AABB anyway, since it can results to better culling? The shape can change a lot depends on weight.

Reply:
I don't think the AABB calculation takes the blend shape weights into account. It uses the bone AABBs and the skeleton position though.